examples/counter/reducers/index.js

```diff
@@ -1 +1,8 @@
-export { default as counter } from './counter';
+import { combineReducers } from 'redux';
+import counter from './counter';
+
+const rootReducer = combineReducers({
+  counter
+});
+
+export default rootReducer;
```

Comment:
No reason in particular. As said, I followed (in this case, just copied) the examples in redux 2 (eg. https://github.com/rackt/redux/blob/master/examples/counter/reducers/index.js)